src/superio/nuvoton/common/early_serial.c

```diff
@@ -83,3 +83,16 @@ void nuvoton_enable_serial(pnp_devfn_t dev, u16 iobase)
 	pnp_set_enable(dev, 1);
 	nuvoton_pnp_exit_conf_state(dev);
 }
+
+u8 nuvoton_read_register(pnp_devfn_t dev, u8 read_reg)
```

Comment:
@piotr-kleins there are still some unused leftovers

Reply:
I left it for debug purpose. It will be deleted in final commit